### PR TITLE
#798 - update management-api

### LINKED ISSUE #798: Update management-api and remove dashboard slices from this repo
Dashboard was moved to `management-api`, let's update version and remove dashboard slices and pages.

Also there are two bugs to fix:
- on start we should not try to read files with `_` prefix and repo configs (`com/artipie/VertxMain.java:197`), these files are credentials, permissions and default storage configuration
- `ArtipieApi` and `DashboardSlice` should read `permissions` sections from `_permissions.yaml` and then pass it to `YamlPermissions`

### PR DESCRIPTION
**Pull request by olenagerasimova**
Closes #798 
I've updated `management-api`, removed dashboard slices and pages, fixed couple of bugs.

### CODE REVIEW

File: src/main/java/com/artipie/VertxMain.java [hunk 1]
```diff
@@ -194,7 +194,7 @@ private void startRepos(final Settings settings, final Metrics metrics) {
         final String yaml = ".yaml";
         final Collection<RepoConfig> configs = storage.list(Key.ROOT).thenApply(
             keys -> keys.stream().map(Key::string)
-                .filter(name -> name.endsWith(yaml))
+                .filter(name -> name.endsWith(yaml) && name.charAt(0) != '_')
```
Comment (olegmoz) [inline src/main/java/com/artipie/VertxMain.java:197]: @olenagerasimova it seems that this change is not really related to other changes in this PR

**Review by olegmoz — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, please see one comment

File: src/main/java/com/artipie/VertxMain.java (hunk 1 above)
Comment (olenagerasimova) [reply src/main/java/com/artipie/VertxMain.java:197]: @olegmoz hm, please, check #798 description - I've described this bug there, so I think it can be make here

Comment (olegmoz) [reply src/main/java/com/artipie/VertxMain.java:197]: @olenagerasimova I see it in the issue now, yet do not understand how these bug fixes are related to replacing existing classes with api-management library. Especially the one with underscore names. If we don't allow names starting with underscore, then with should document it

Comment (olenagerasimova) [reply src/main/java/com/artipie/VertxMain.java:197]: @olegmoz dashboard with `org` layout cannot be used while any of these bugs are valid - I faced them trying to start artipie and access any dashboard page locally. 
I'd suggest to properly deal with underscore names (or catching exceptions with invalid repo configs, whatever we decide) in the separate issue/PR.

Comment (olegmoz) [reply src/main/java/com/artipie/VertxMain.java:197]: @olenagerasimova got it, thanks

**Review by olegmoz — APPROVED** [review]
